config/webpack.config.js

```diff
@@ -185,6 +185,8 @@ const configGenerator = (buildOptions, apps) => {
       new webpack.DefinePlugin({
         __BUILDTYPE__: JSON.stringify(buildOptions.buildtype),
         __API__: JSON.stringify(buildOptions.api),
+        // eslint-disable-next-line import/no-unresolved
+        __MEGAMENU_CONFIG__: JSON.stringify(require('../.cache/megamenu.json')),
```

Comment:
@jbalboni Added this change to just compile the data directly into Webpack

Reply:
I just realized that you'll have to restart the watch task for the MegaMenu data to refresh, whereas passing the path to the cache file would allow it to rebuild. I think this is okay though, because the MegaMenu isn't changed often.

Reply:
Ah, yeah. Does it work if you you make `__MEGAMENU_CONFIG__` the file name instead of the content and use it in an import or require in the code? Either way, yeah, it's not likely to be an issue.

Reply:
I think it would work, but it's a little confusing because the path to the `.cache` directory would be different based on where it's imported

Reply:
Oh, right, never mind.